packages/select/src/components/select/_select.scss

```diff
@@ -3,29 +3,41 @@
 
 @import "../../common/variables";
 
+$select-padding: $pt-grid-size * 0.5;
+
 .#{$ns}-select-popover {
   .#{$ns}-popover-content {
     // use padding on container rather than margin on input group
     // because top margin leaves some empty space with no background color.
-    padding: $pt-grid-size * 0.5;
+    padding: $select-padding;
   }
 
   .#{$ns}-input-group {
     margin-bottom: 0;
   }
 
   .#{$ns}-menu {
+    // offset the padding added by the select componenent so menu dividers can remain full width
+    margin-left: -$select-padding;
+    margin-right: -$select-padding;
     max-height: $select-popover-max-height;
     max-width: $select-popover-max-width;
     overflow: auto;
     padding: 0;
 
     &:not(:first-child) {
       // adjust padding to account for that on .#{$ns}-popover-content above
-      padding-top: $pt-grid-size * 0.5;
+      padding-top: $select-padding;
     }
   }
 
+  .#{$ns}-menu-item {
```

Comment:
I wonder if there may be other classes we need to target with adding back the margin we take away above...

From what I could tell from the docs it seems like only elements that get this `bp-menu-item` class are intended to go into a menu, but I suppose it may be allowed by the API to develop custom menu items that may now shift around with this change?

Reply:
ahh, I think this was a good note... pushing to make the divider and menu items not be a special case got me to bring back the padding on the actual menu, which allows the divider to keep it's normal negative margin

Reply:
Nice!